src/ui/pages/VerifySeedPhrase/VerifySeedPhrase.test.tsx

```diff
@@ -207,25 +193,6 @@ describe("Verify Seed Phrase Page", () => {
     );
 
     fireEvent.click(continueButton);
-
-    const seedPhraseString = initialState.seedPhraseCache.seedPhrase;
-    const entropy = Addresses.convertToEntropy(seedPhraseString);
-    const Bech32XPrv = Addresses.entropyToBip32NoPasscode(seedPhraseString);
-    expect(Addresses.convertToEntropy).toBeCalledWith(seedPhraseString);
-    expect(Addresses.entropyToBip32NoPasscode).toBeCalledWith(entropy);
-    expect(Addresses.bech32ToHexBip32Private).toBeCalledWith(Bech32XPrv);
-
-    expect(SecureStorage.set).toBeCalledWith(
-      KeyStoreKeys.IDENTITY_ROOT_XPRV_KEY,
-      rootKeyHex
-    );
-
-    await waitFor(() =>
-      expect(SecureStorage.set).toBeCalledWith(
-        KeyStoreKeys.IDENTITY_ENTROPY,
-        entropy
-      )
-    );
```

Comment:
I guess this test should've been updated before - can you add the check that `KeyStoreKeys.SIGNIFY_BRAN` was stored here?